middleware/githubConfig.js

```diff
@@ -4,19 +4,18 @@ const config = require('painless-config')
 const githubMiddleware = require('./github')
 const githubRoute = require('../routes/auth')
 const permissions = require('./permissions')
+const memoryCache = require('../providers/caching/memory')
 
 const defaultOptions = {
   clientId: config.get('AUTH_GITHUB_CLIENT_ID'),
   clientSecret: config.get('AUTH_GITHUB_CLIENT_SECRET'),
   token: config.get('CURATION_GITHUB_TOKEN'),
   org: config.get('AUTH_GITHUB_ORG') || 'clearlydefined',
-  timeouts: {
-    info: 10 * 60 // 10 mins
-  },
   permissions: {
     harvest: [config.get('AUTH_HARVEST_TEAM') || 'harvest-dev'],
     curate: [config.get('AUTH_CURATION_TEAM'), 'curation-dev']
-  }
+  },
+  cache: memoryCache({ defaultExpirationSeconds: 10 * 60 /* 10 mins */ })
```

Comment:
yeah, so down here (actually L23) perhaps we pass along the cache in addition to the declarative options rather than adding the cache into options.